### DISCUSSION TITLE
Export CSVs (or other lists) of collections?

### DISCUSSION THREAD

**Discussion by gabestein** [knowledgefutures/pubpub#1341] [Ideas]
We had a user suggest this idea, which I think is pretty cool! Anyone else interested in exporting lists of collections for various purposes? If so, it would be helpful to know what you'd use it for, and if you have any thoughts on formatting.

I should note that this is somewhat separate from the idea of exporting collection content — e.g., a single ePub containing all the Pubs in the collection — which is something we'd also like to do at some point.

**Comment by tosteiner** [comment]
@gabestein Gabe, I realise this might be a bit late, but I would certainly be interested in the kind of export option you describe (plus a csv-based list of all pubs published in a given community, with available metadata, if at all possible) - these would really be useful for creating output activity reports etc. 👍🏾
